docs/fastdds/dynamic_types/dynamic_types.rst

```diff
@@ -1,13 +1,13 @@
 .. _dynamic-types:
 
-Dynamic Topic Types
-===================
+XTypes 1.3
+==========
 
-.. _DDS-XTypes V1.2: http://www.omg.org/spec/DDS-XTypes/1.2
+.. _DDS-XTypes V1.3: http://www.omg.org/spec/DDS-XTypes/1.3
```

Comment:
This URL cannot be access.

```suggestion
.. _DDS-XTypes V1.3: https://www.omg.org/spec/DDS-XTypes/1.3
```

Reply:
fixed in [e6714a3](https://github.com/eProsima/Fast-DDS-docs/pull/730/commits/e6714a3c5febe9d35a953b5c4f57fb084ed94a81)